spec/monotonic_tick_count_spec.rb

```diff
@@ -0,0 +1,125 @@
+require "spec_helper"
+
+describe MonotonicTickCount do
+  it "should have a version number" do
+    expect(::MonotonicTickCount::VERSION).to be
+  end
+
+  context "#initialize" do
+    it "should snapshot the current tick counter when constructed with no arguments" do
+      expect(Process).to receive(:clock_gettime).with(Process::CLOCK_MONOTONIC) { 1234.012345 }
+      tick_count = MonotonicTickCount.new
+      expect(tick_count.tick_count_f).to eq(1234.012345)
+    end
+
+    it "should have a copy constructor" do
+      tick_count1 = MonotonicTickCount.new
+      tick_count2 = MonotonicTickCount.new(tick_count1)
+      expect(tick_count2.tick_count_f).to eq(tick_count1.tick_count_f)
```

Comment:
This can be simplified to
```
expect(tick_count1).to eq(tick_count2)
```

Reply:
True, it can now, now that the class includes `Comparable`. Done.